cmd/cedana-helper/main.go

```diff
@@ -132,7 +117,7 @@ func runCommand(command string, args ...string) error {
 }
 
 func startDaemon() (int, error) {
-	cmd := exec.Command(daemonCommand, "daemon", "start&")
+	cmd := exec.Command("../../build-start-daemon.sh")
```

Comment:
no issues w/ systemd here? happy to add a flag to the script that controls whether or not the daemon is started as a systemd daemon

Reply:
No issues with running as a systemd daemon since this is technically on the ec2 instance so we get access to systemctl